forms/contributor-signup.md

```diff
@@ -0,0 +1,40 @@
+Google form link: to be created
+
+# Mentored Sprint at PyCon US: Contributor Sign Up
+
+Please fill in this form if you would like to be mentored during the sprint.
+
+For more information, please check: https://us.pycon.org/2019/hatchery/mentoredsprints/
+
+1. Email address (required)
+
+2. Your name (required)
+
+3. Are you a member of underrepresented group?
```

Comment:
I'm not sure if this should be asked at all?

Reply:
:thinking: I believe there is value in collecting information like that. The aim of the sprint is to introduce new contributors to open source but we do  want to mainly target underrepresented groups. And collecting these data will help us know whether we achieved our goal. 

I would regret, however, if a person of a represented group that does not feel comfortable attending the sprints not taking this opportunity to learn about open source because the felt intimidated by the question, or even people of underrepresented feeling offended (:woman_shrugging: ). 

For these reasons I would suggest a small modification:

```
- questions 3 (text and not a required field): Preferred pronouns 
```
and later on like question 10:
```
- question 10: Do you consider yourself a member of an underrepresented group?
- yes
- no
- prefer not to say
```

what do you think?

Reply:
I generally prefer the wording that Nikoleta suggested (usually followed by an 'if so, please feel free to elaborate')

Reply:
That said it might be useful if we plan to do an impact report or something similar to what  the women in ML did for [https://reshamas.github.io/impact-report-for-wimlds-scikit-learn-sprints/](https://reshamas.github.io/impact-report-for-wimlds-scikit-learn-sprints/)

Reply:
A little off topic, but if someone writes their preferred pronouns, how will we keep track of that and match it to the person? If someone fills in their pronouns on the form for an event and isn't called by those pronouns at the event itself, that might not be the best experience.

Reply:
Following on the pronouns:
- PyCon provides pronoun ribbons for all the attendees, so if an attendee has a pronoun ribbon that covers it for us. However, if the person chooses not to use one or did not know there were ribbons available, shall we be prepared and have stickers/something with other options or pronouns? (i.e. nor pronoun just my name, ask me, etc).

- I suggest changing `- questions 3 (text and not a required field): Preferred pronouns` for `- questions 3 (text and not a required field): Used pronouns`